README.md

```diff
@@ -72,9 +72,9 @@ Face detection models identify and/or recognize human faces in images. Some more
 
 |Model Class |Reference |Description |
 |-|-|-|
```

Comment:
consider shortening to "body_analysis" instead of "Body_Face_and_Gesture_Analysis"